BootloaderCorePkg/Tools/GenCfgData.py

```diff
@@ -1538,6 +1538,32 @@ def GenerateDataIncFile (self, DatIncFileName, BinFile = None):
 
         return 0
 
+    def CheckCfgData (self):
+        # Check if CfgData contains any duplicated name
+        def  AddItem (Item, ChkList):
+            Name = Item['cname']
+            if Name in ChkList:
+                return Item
+            if Name not in ['Dummy', 'Reserved', 'CfgHeader', 'CondValue']:
```

Comment:
Can we ignore fields named 'Rsvd*' also? There might be multiple fields with name 'Rsvd' that might be a hit here.

Reply:
Let us scandalize the keyboards used in CFGDATA.  Rsvd* normally is for field in a sub region.  So it could be duplicated within different sub region scope.   I have verified the current change worked fine for QEMU and APL.